relayer/middleware/providers.middleware.ts

```diff
@@ -108,6 +109,11 @@ const defaultSupportedChains = {
         "https://eth-goerli.g.alchemy.com/v2/mvFFcUhFfHujAOewWU8kH5D1R2bgFgLt",
       ],
     },
+    [CHAIN_ID_SEPOLIA]: {
+      endpoints: [
+        "https://eth-sepolia.g.alchemy.com/v2/mvFFcUhFfHujAOewWU8kH5D1R2bgFgLt",
```

Comment:
should we set a public one instead?

Reply:
This works, but you can add alternate endpoints I guess.